code/web-app-ri/VotingWeb/VotingWeb.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Web">
```

Comment:
No nuget package updates?

Reply:
Updated to the latest in both solutions.

Reply:
Thanks!